### ISSUE TITLE
Cannot uninstall system app

### ISSUE THREAD

**Issue by RigacciOrg** [TeamAmaze/AmazeFileManager#840]
I installed Amaze 3.2.1 from F-Droid into an Android 6.0 rooted chinese phone. It can uninstall user Apps installed via Google Play and via F-Droid without problems, but it fails when I try to uninstall **com.android.service**, which is a spyware installed by the chinese manufacturer (at least accordingly with Google Play Protect and some security sites).
When I tap the "uninstall" option, the Amaze app warns about removing a system app, the it says "done", but the app is still there, indeed. I **DO NOT** have any extra tools installed, like busybox.
It seems that the backup option instead is working well with system app too.

**Comment by EmmanuelMess** [comment]
It seems to be very specific to that app...

**Comment by RigacciOrg** [comment]
What is the Amaze recipe to uninstall a system app? I wish to try the same recipe from the command line. I'm quite unable to read Android sources :-(
Does the uninstall operation leave some tracks in system logs? I'm browsing them, but I cannot figure any clue.

**Comment by EmmanuelMess** [comment]
@VishalNehra should know.

**Comment by VishalNehra** [comment]
Are you rooted? You must be rooted to uninstall system apps.

**Comment by RigacciOrg** [comment]
Yes, I'm rooted. But I don't have any extra tools, like busybox, if it is relevant.

**Comment by RigacciOrg** [comment]
Hi, I have some hints for debug.

I tried OpenAPK version 1.2 (installed from F-Droid), which should be able to remove system Apps too; it failed at least partially. Looking at SuperSU popup messages, I see that it removed one .apk file (so the program seems almost killed), but it leaved an .odex file. The two files are:
`
/system/priv-app/com.android.service-9002_0711/com.android.service-9002_0711.apk
/system/priv-app/com.android.service-9002_0711/oat/arm/com.android.service-9002_0711.odex
`
From the command line I remounted /system as rw, removed the entire directory, and remounted ro. Reboot to be safe, and the malicious system app is gone.
